.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,52 +1,60 @@
 ---
-name: Bug report
-about: Report a bug in this library
+name: Library-specific Bug Report
+about: Report a problem with this library that needs to be fixed.
 title: ''
-labels: bug
+labels: bug, triage
 assignees: ''
 
 ---
 
-<!--
-This form is for bug reports for this library only.
-For general, language-agnostic questions and troubleshooting about the [Google Ads API](https://developers.google.com/google-ads/api/docs/start), please ask/look for answers [here](https://groups.google.com/g/adwords-api).
+<!-- PLEASE READ
+This form is for bug reports specific to this library.
+
+For troubleshooting tips, see: https://developers.google.com/google-ads/api/docs/best-practices/troubleshooting.
+
+For issues that are related to the Google Ads API itself and not specific to this library, please reach out to one of our support channels: https://developers.google.com/google-ads/api/support.
 
 Please check existing issues to see if your bug has already been reported before creating a new issue. We encourage you to add any relevant information to the existing issue if one already exists.
 -->
 
 [DO NOT INCLUDE ANY PERSONAL OR SENSITIVE DATA - MAKE SURE TO REDACT CONTENT WHEN NECESSARY]
 
-**What versions of the client library and Google Ads API are you using?**
+**Your client library and Google Ads API versions:**
 
-<!--
-- Client library (format: `vX.Y.Z`): the exhaustive list of all released versions of the client library is in the repository [releases' page](https://github.com/googleads/google-ads-php/releases).
-- Google Ads API (format: `VX`): you can find the major version of the Google Ads API used by checking the namespace of your `GoogleAdsClient`, e.g., `Google\Ads\GoogleAds\Lib\V6\GoogleAdsClient`.
--->
+* Client library version: `vX.Y.Z`
 
-**What environment are you using?**
+<!-- This should be a released version, see https://github.com/googleads/google-ads-php/releases. -->
```

Comment:
Is this comment needed?

Reply:
It is easy to confuse the client library version for the Google Ads API version, this aims to clarify.

Reply:
I think having "Client library version" and "Google Ads API version" as separate points here, as you have them, should serve to clarify that.

Reply:
I agree that it makes it clear that those are two different things but it does not give any hint on how to identify the one being used though.

Reply:
From my experience, this is really needed. Beginners are mostly confused by several versions we use.
Even with the old template, people still get confused.